.github/ISSUE_TEMPLATE/NEW_COMPONENT.yml

```diff
@@ -0,0 +1,53 @@
+name: New Component
+description: Provide details to help define a new component for the system
+labels: [feature request, pending triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **⚠️ This is a public repository. Please be aware of sensitive information and prefer referencing internal links.**
+
+        Please leverage this template to provide as much details about the visual and functional requirements for this new component. Doing so will help use communicate and collborate on a solution effectively.
+
+  - type: text
+    attributes:
+      label: Design File
+      description: Provide a Figma link as a reference
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Provide as much details as possible about the visual and functional requirements.
+      placeholder: |
+        <!--
+          Consider UI/UX like affordance, user interaction, and different states
+        -->
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Research
+      description: |
+        How similar is this new component to any existing component?
+      placeholder: |
+        <!--
+          Link to any existing components from Maker or other design systems (Material, Apple, Carbon, etc.) and describe their similarities
```

Comment:
Add links to common design systems to make it easier to find.

Some contributors may not know the common design systems, or what to search, if they're new to UI.

Reply:
That's fair, but a list of links to a few design systems starts to feel prescriptive and non-exhaustive. There's dozens of great systems out there and as placeholder text you can't click on any links. I agree it's worth potentially a bit more guidance, but I don't have a better alternative idea.

Reply:
I think this section sounds prescriptive because it comes off as a task and doesn't explain the goal (eg. why do we care about other design systems?).

I'm actually not sure what your intention is here, because I think it would vary dramatically depending on whether this template should be filled out by an engineer tasked with implementation, or the component designer.

Making the intention and end-goal clear would give the contributor more creative freedom.

----

FYI As an engineer that implemented many components, I didn't often get value from learning what other UI libraries are doing. I might even go as far as saying it's distracting from listening to our own users and use-cases.

Reply:
Several recent components we've referenced Orbit implementations since we have similar problems and solutions. 

From an engineering standpoint, I do think it provides value to research a little bit around the conventions of reusable components in other systems as it helps compare and/or contrast implementation details that often get overlooked until implementation starts. 

Knowing up front what visual / functional requirements may be needed or missed in the original design provides everyone an opportunity to align and iterate on those requirements. Having other real-world examples can speed up that process and make communication easier.

If any new component is so unique that there's no relevant comparison then the answer to this question can simply be "None"